requirements.yml

```diff
@@ -1 +1,2 @@
 ---
```

Comment:
Some information about this package being required would be nice in the readme (Unless it is there and I simply missed it)